ex5/ex5.c

```diff
@@ -16,6 +17,20 @@ char* msg3 = "hello world #3";
 int main()
 {
     // Your code here
-    
+    int fds[2];
+    char buffer[128];
+
+    int child = fork();
+
+    if (child < 0) {
+      printf("Cannot fork a child\n");
+    } else if (child == 0) {
+      write(fds[1], msg1, strlen(msg1));
```

Comment:
Here, you're only writing one of the messages to the parent process. Make sure to write all three to the parent.